### ISSUE TITLE
Regrowth not used anymore

### ISSUE THREAD

**Issue by Bestoriop** [Zebouski/QuickHeal#2]
As a druid, quickheal never uses Regrowth any more. That's a big problem !

**Comment by Zebouski** [comment]
I don't have a druid to test.

Quickheal has a HPM efficient mode, and a HPS output mode that sacrifices mana efficiency. I assume you have tried the HPS mode? Does it not use regrowth there?

**Comment by Bestoriop** [comment]
Nvm, regrowth is used in High HPS mode. 

I don't understand why the threshold was removed for them ?

**Comment by Zebouski** [comment]
what do you mean threshhold? I don't understand the current issue now

**Comment by Bestoriop** [comment]
For priest/paladin/shaman you have a threshold you can modify to chose at what % the high hps spell will be used. Druids don't have that anymore.

**Comment by Bestoriop** [comment]
It's problematic because as a resto druid, you may want to use regrowth even if you use healing touch most of the time

**Comment by Bestoriop** [comment]
If you don't have a druid i can test for you. My druid is 43 btw

**Comment by Zebouski** [comment]
> It's problematic because as a resto druid, you may want to use regrowth even if you use healing touch most of the time

yup, you need to keybind the quickheal toggle if you want to change up modes on the fly. QH doesnt automate that, thats the player's decision to swap HPS and HPM modes.

on my priest, this switches me from flashing to the slow heal. on pally from holy light to flash. i dont see how its any diff for your druid.

**Comment by Bestoriop** [comment]
I understand how it works now but why did you remove it ?

**Comment by Bestoriop** [comment]
I mean the main goal of quickheal is make the decision faster. When i'm in a 40 players raid, if one of those players drop under 20% while 39 others are at 70-90%, i need quickeal to use regrowth to heal that guy as fast as possible. I won't norice that fast enough myself.

**Comment by Bestoriop** [comment]
Did anyone asked you to remove that feature for druids ?

**Comment by Zebouski** [comment]
> I understand how it works now but why did you remove it ?

read the description, im not the dev, im merely rehosting this addon from a dev who took it down

i'll leave valid bug reports/feature requests up for any dev who wants to fulfil them

so anyway, i dont think QH ever had support for this. not this fork of it at least.
